base/array.jl

```diff
@@ -1533,7 +1533,8 @@ function findnext(testf::Function, A, start::Integer)
     l = endof(A)
     i = start
     while i <= l
-        if testf(A[i])
+        @inbounds ai = A[i]
```

Comment:
this isn't inbounds on the first iteration

Reply:
i < 1 kills us? Sorry, will revert.